src/lib/components/prompt/PromptSuggestion.svelte

```diff
@@ -97,7 +97,7 @@
 		{:else if applyingSuggestion}
 			<Spinner />
 		{:else}
-			<Button {disabled} onclick={accept}>Apply</Button>
+			<Button classNames="h-8" {disabled} onclick={accept}>Apply</Button>
```

Comment:
why does this need a height? Shouldn't the items-center center this?

Reply:
just to make it slightly shorter than other buttons I thought it looked a bit nicer.

Reply:
ah ooops I thought this was dismiss, never mind